collect_app/src/androidTest/java/org/odk/collect/android/support/pages/FormEntryPage.java

```diff
@@ -54,6 +54,15 @@ public FormEntryPage swipeToNextQuestion(int repetitions) {
         return this;
     }
 
+    public FormEntryPage swipeToNextRepeat(String repeatLabel, int repeatNumber) {
+        tryAgainOnFail(() -> {
+            onView(withId(R.id.questionholder)).perform(swipeLeft());
```

Comment:
We might need to move all of our `swipeToX` helpers to this kind of style. It's definitely safer for us to have assertions with actions like gestures that can be performed on any screen.